src/main/java/stream/flarebot/flarebot/database/SQLController.java

```diff
@@ -25,7 +25,7 @@
         dataSource.setDatabaseName("flarebot");
         dataSource.setPort(3306);
         dataSource.setPortNumber(3306);
-        dataSource.setServerName("127.0.0.1");
+        dataSource.setServerName("192.168.254.100");
```

Comment:
?

Reply:
@weeryan17 my god

Reply:
you saw nothing!!!!. also yegh my bad. that was for my mysql stuff

Reply:
xDD